ci-operator/jobs/openshift/osde2e/openshift-osde2e-master-periodics.yaml

```diff
@@ -1100,7 +1147,7 @@ periodics:
       - name: ADDON_IDS
         value: codeready-workspaces-qe
       - name: ADDON_TEST_HARNESSES
-        value: quay.io/crw/osd-e2e:latest
+        value: quay.io/crw/osd-e2e:2.1.1-nightly
```

Comment:
Shouldn't we put here just "nightly" tag? 
Because this way we would have to modify this line everytime we promote new CRW version to our qe addon.

Reply:
Yes, you're right, I'll change that.